client/my-sites/email/titan-setup-thank-you/index.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ThankYou } from 'calypso/components/thank-you';
+import theme from 'calypso/my-sites/marketplace/theme';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getTitanEmailUrl } from 'calypso/lib/titan';
+import Gridicon from 'calypso/components/gridicon';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+
+type TitanSetupThankYouProps = {
+	domainName: string;
+	emailAddress?: string;
+};
+
+export const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
+	const currentRoute = useSelector( getCurrentRoute );
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const translate = useTranslate();
+	const thankYouImageSrc = '/calypso/images/upgrades/thank-you.svg';
+
+	const { domainName, emailAddress = `youremail@${ domainName }` } = props;
+
+	const emailManagementPath = emailManagement( selectedSiteSlug, domainName, currentRoute );
+
+	const thankYouImage = {
+		alt: 'Thank you',
+		src: thankYouImageSrc,
+	};
+
+	const titanThankYouSection = {
+		sectionKey: 'titan_whats_next',
+		sectionTitle: translate( 'What’s next?' ),
+		nextSteps: [
+			{
+				stepKey: 'titan_whats_next_view_inbox',
+				stepTitle: translate( 'Access your inbox' ),
+				stepDescription: translate( 'Access your email from anywhere with our built-in webmail.' ),
+				stepCta: (
+					<FullWidthButton href={ getTitanEmailUrl( emailAddress ) } primary>
+						{ translate( 'Go to Inbox' ) }
+						<Gridicon icon="external" />
```

Comment:
We should fix the empty margin between the label and the icon:

<img width="255" alt="screenshot" src="https://user-images.githubusercontent.com/594356/127190101-cf29d9b6-306d-4d59-81e0-55062ef1e9b6.png">